util.py

```diff
@@ -100,7 +104,10 @@ def get_user_points() -> Dict[str, int]:
       commit_username, = match.groups()
 
       if commit_username in points:
-        points[commit_username] += 1
+        points[commit_username]['merge'] += 1
+
+  for user in points:
+    points[user]['total'] = sum(points[user].values())
```

Comment:
The more I think about it, the more overkill this seems. If you feel strongly about it, I'll implement it, but otherwise I'm inclined to just keep it sorta simple.

Reply:
I don't feel strongly about this

Reply:
My objection is that it allows the total to be updated inappropriately, or more likely, fail to be updated appropriately, at some point in the future.

Reply:
That's a valid concern.

Would you be ok with merging this in as it stands, and me opening a follow-up PR to address it?

Reply:
Not a huge deal either way.

Reply:
Since you need re-approvals anyway, maybe just do it properly here?

Reply:
By do it properly, do you mean making it a proper object?

Reply:
Yes.  (Or making a function that takes a points dict and computes a total, and using that every time you need someone's total points.)